kubernetes/kube-state-metrics-deployment.yaml

```diff
@@ -1,4 +1,5 @@
 apiVersion: apps/v1beta2
+# for Kubernetes versions after 1.9.0 could use apps/v1
```

Comment:
`for Kubernetes versions after 1.9.0 could use apps/v1` -> `Kubernetes versions after 1.9.0 should use apps/v1`